src/network/network.cpp

```diff
@@ -1947,6 +1951,14 @@ void Network::Client_Handle_GAMEINFO(NetworkConnection& connection, NetworkPacke
 		ServerProviderWebsite = json_stdstring_value(json_object_get(jsonProvider, "website"));
 	}
 	json_decref(root);
+
+	// Display server greeting if one exists
+	const char* greeting = network_get_server_greeting();
+	if (strcmp(greeting, "") != 0) {
+		char greeting_formatted[256];
```

Comment:
Some hosts may choose to provide a longer message than this limit. Few bytes will be also taken by formatting, you should limit the user-defined greeting to something that would fit, say 240 chars and provide warning to user if this gets truncated. See `safe_str*` functions, they already do that.